pulp_ansible/app/tasks/collections.py

```diff
@@ -534,6 +535,16 @@ async def _add_collection_version(self, api_version, collection_version_url, met
         await self.parsing_metadata_progress_bar.aincrement()
         await self.put(d_content)
 
+        if signatures := metadata.get("signatures"):
```

Comment:
in pulp-container we've added a boolean option which can be specified during sync `signed_only`. In case it's enabled it would skip any content that has no signature, does it make sense to add it here as well?

Reply:
makes sense as we are going to have sort of `SYNC_ONLY_SIGNED_CONTENT=True` for Private Hub config, we were thinking about doing that per remote and then pass it to the sync task.